ci/wasm_compile.py

```
@@ -73,13 +79,13 @@ def main() -> int:
         print(f"Note: Could not verify filter compatibility ({type(e).__name__})")
 
     # Print what we're going to do
-    if args.run:
+    if run_browser_check:
         _print_panel(
             "FastLED WASM Build Pipeline",
             [
                 f"Will:",
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)